src/foam/nanos/u2/navigation/SignIn.js

```diff
@@ -126,7 +131,10 @@ foam.CLASS({
                     this.user.copyFrom(updatedUser);
                     this.nextStep();
                   }).catch((err) => {
-                    this.notify(err.message || this.ERROR_MSG, 'error');
+                    this.ctrl.add(this.NotificationMessage.create({
```

Comment:
why not use the notify method here?

Reply:
so for sign in and sign up the user is not logged in yet. So I have to just display a NotificationMessage as we need the logged in user to send notification messages through the notificationDAO.

Reply:
ok.